### DISCUSSION TITLE
improve the left sidbar

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2812] [Ideas]
Hello guys,

when we create quite complexe elements - and not so complexe - the left sidbar is just frustrating.

![image](https://user-images.githubusercontent.com/3023526/135431250-9085fbdf-7742-4170-aa5b-708dceefec01.png)

I cannot see the element at the far right.
How can we improve this?

**Comment by mjashanks** [comment]
Agree - also discussed here: https://github.com/Budibase/budibase/discussions/2158
